features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/jagg/jaggery_acs.jag

```diff
@@ -52,6 +52,7 @@
         ssoRelyingParty.setProperty("signRequests", String(site.ssoConfiguration.signRequests));
         ssoRelyingParty.setProperty("nameIdPolicy", site.ssoConfiguration.nameIdPolicy);
         ssoRelyingParty.setProperty("assertionEncryptionEnabled", site.ssoConfiguration.assertionEncryptionEnabled);
+        ssoRelyingParty.setProperty("loginUserNameAttribute", site.ssoConfiguration.loginUserNameAttribute);
     }
```

Comment:
What is the default value for site.ssoConfiguration.loginUserNameAttribute, in the site.json, We don't have default value in site.json or do we?

Reply:
In case loginUserNameAttribute is not set in site.json, getUsernameFromAssertion() reads username from NameID of assertion subject. 
https://github.com/wso2/carbon-apimgt/pull/5080/files#diff-3cc49a603ac9062db1b85f4b8e81f27dR515